src/tool_libinfo.c

```diff
@@ -68,23 +68,81 @@ static struct proto_name_tokenp {
   {  NULL,      NULL         }
 };
 
+bool feature_altsvc = FALSE;
+bool feature_brotli = FALSE;
+bool feature_hsts = FALSE;
+bool feature_http2 = FALSE;
+bool feature_http3 = FALSE;
+bool feature_libz = FALSE;
+bool feature_ntlm = FALSE;
+bool feature_ntlm_wb = FALSE;
+bool feature_spnego = FALSE;
+bool feature_ssl = FALSE;
+bool feature_tls_srp = FALSE;
+bool feature_zstd = FALSE;
+
+static struct feature_name_presentp {
+  const char   *feature_name;
+  bool         *feature_presentp;
+  int           feature_bitmask;
+} const maybe_feature[] = {
+  /* Keep alphabetically sorted. */
+  {"alt-svc",        &feature_altsvc,     CURL_VERSION_ALTSVC},
+  {"AsynchDNS",      NULL,                CURL_VERSION_ASYNCHDNS},
+  {"brotli",         &feature_brotli,     CURL_VERSION_BROTLI},
+  {"CharConv",       NULL,                CURL_VERSION_CONV},
+  {"Debug",          NULL,                CURL_VERSION_DEBUG},
+  {"gsasl",          NULL,                CURL_VERSION_GSASL},
+  {"GSS-API",        NULL,                CURL_VERSION_GSSAPI},
+  {"HSTS",           &feature_hsts,       CURL_VERSION_HSTS},
+  {"HTTP2",          &feature_http2,      CURL_VERSION_HTTP2},
+  {"HTTP3",          &feature_http3,      CURL_VERSION_HTTP3},
+  {"HTTPS-proxy",    NULL,                CURL_VERSION_HTTPS_PROXY},
+  {"IDN",            NULL,                CURL_VERSION_IDN},
+  {"IPv6",           NULL,                CURL_VERSION_IPV6},
+  {"Kerberos",       NULL,                CURL_VERSION_KERBEROS5},
+  {"Largefile",      NULL,                CURL_VERSION_LARGEFILE},
+  {"libz",           &feature_libz,       CURL_VERSION_LIBZ},
+  {"MIME",           NULL,                0},
```

Comment:
This is not an actual feature is it?

Reply:
No it isn't. This is a leftover from local tests. Do not harm, but as useless, removed now.